app/views/home/_fandoms.html.erb

```diff
@@ -0,0 +1,10 @@
+<ul class="menu">
```

Comment:
I think you can take the class off in addition to the role -- it doesn't seem to be used for the styling and doesn't make much sense here anyway.